MKESocial/app/src/main/res/layout/activity_create_event.xml

```diff
@@ -8,94 +9,279 @@
         android:layout_height="wrap_content"
         android:gravity="center_horizontal"
         android:orientation="vertical"
-        android:paddingBottom="32dp"
-        android:paddingEnd="32dp"
-        android:paddingLeft="32dp"
-        android:paddingRight="32dp"
-        android:paddingTop="24dp">
-
-        <EditText
-            android:id="@+id/edit_title"
+        android:paddingBottom="16dp"
+        android:paddingEnd="16dp"
+        android:paddingLeft="8dp"
+        android:paddingRight="8dp"
+        android:paddingTop="16dp">
+
+        <GridLayout
+            android:layout_width="match_parent"
+            android:layout_height="match_parent"
+            android:orientation="horizontal">
+
+            <ImageView
+                android:id="@+id/imageView10"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:layout_gravity="bottom"
+                android:layout_marginBottom="12dp"
+                app:srcCompat="@mipmap/ic_title_t" />
```

Comment:
The build is actually broken because the image is really <tt>ic_title_tt</tt>. Or did you intend to name it with one t?

Reply:
two t's, shoot it didn't push correctly :(
Thank you for catching that!

Reply:
Apparently you can't name icon mipmap with caps, and github ignores case sensitivity, so changing case didn't register in git.... so changed names... and here we are xD